tests/test_default.py

```diff
@@ -0,0 +1,48 @@
+
+from testinfra.utils.ansible_runner import AnsibleRunner
+
+
+testinfra_hosts = AnsibleRunner('.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_directories(File):
```

Comment:
Please change used class from `File` to `host` and use newer testinfra fixtures. Example can be found in [ansible-mongo tests](https://github.com/SoInteractive/ansible-mongodb/blob/master/tests/test_default.py)